src/apis/schema/mission.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @description 미션 스키마
+ * @param missionId
+ * @param name - 미션 이름
+ * @param content - 미션 내용
+ * @param category - 미션 카테고리
+ * @param visibility - 미션 공개여부
+ * @param sort - 미션 정렬 값
+ */
+export interface MissionType {
+  missionId: number;
+  name: string;
+  content: string;
+  category: string;
+  visibility: string;
```

Comment:
category 와 visibility 는 서버에서 Enum 으로 관리되고 있는 것 같아요 (stoplight 에서 Allowed values 가 있는 것들은 Enum으로 관리되어 있는 것 같습니다)

```suggestion
  export enum MissionCategory {
  STUDY = 'STUDY',
  EXERCISE = 'EXERCISE',
  READING = 'READING',
  WRITING = 'WRITING',
  PROJECT = 'PROJECT',
  WATCHING = 'WATCHING',
  ETC = 'ETC',
}

export enum MissionVisibility {
  ALL = 'ALL',
  FOLLOWER = 'FOLLOWER',
  NONE = 'NONE',
}

visibility : MissionVisibility
category : MissionCategory
```